dev/DynamicDependency/M.AM.DD.CreatePackageDependencyOptions.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "Microsoft.ApplicationModel.DynamicDependency.CreatePackageDependencyOptions.g.h"
+
+#include "winrt_namespaces.h"
+
+namespace winrt::Microsoft::ApplicationModel::DynamicDependency::implementation
+{
+    struct CreatePackageDependencyOptions : CreatePackageDependencyOptionsT<CreatePackageDependencyOptions>
+    {
+        CreatePackageDependencyOptions() = default;
+
+        winrt::PackageDependencyProcessorArchitectures Architectures();
+        void Architectures(Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures const& value);
+        bool VerifyDependencyResolution();
+        void VerifyDependencyResolution(bool value);
+        winrt::PackageDependencyLifetimeArtifactKind LifetimeArtifactKind();
+        void LifetimeArtifactKind(Microsoft::ApplicationModel::DynamicDependency::PackageDependencyLifetimeArtifactKind const& value);
+        hstring LifetimeArtifact();
+        void LifetimeArtifact(hstring const& value);
+
+    private:
+        winrt::PackageDependencyProcessorArchitectures m_architectures = winrt::PackageDependencyProcessorArchitectures::None;
```

Comment:
prefer {} for initializers; it also lets you use empty list to help save brain space when you're initializing to the default/0 of a POD.

Reply:
I usually do. Thanks for spotting this.

P.S. For the home audience [here's the guideline for {}](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)